legend-depot-store-mongo/src/main/java/org/finos/legend/depot/store/mongo/entities/EntitiesMongo.java

```diff
@@ -170,169 +141,81 @@ private Bson combineDocument(StoredEntity entity)
     @Override
     public Optional<Entity> getEntity(String groupId, String artifactId, String versionId, String path)
```

Comment:
am sure there are many methods that can be moved up to the abstract entity mongo

Reply:
made the change now